README.md

```diff
@@ -99,15 +100,14 @@ There are several directories within the repository to maintain an organised pro
 #### **IMPORTANT**
 >Clicking on the link provided will redirect you to the README in that directory which provides more information
 
-- `archive` - Contains scripts that archive month-old data in the database to avoid over-inflation of data storage. [**Click here**](./archive/README.md).
-- `dashboard` - Contains all the scripts involved in creating and hosting the dashboard. [**Click here**](./dashboard/README.md).
-- `database` - Contains the schema used to create the RDS hosted on AWS as well as any auxiliary functions. [**Click here**](./database/README.md).
+- [`archive`](./archive/README.md). - Contains scripts that archive month-old data in the database to avoid over-inflation of data storage. 
```

Comment:
What happened here?

Reply:
I just made it so the word is hyperlinked.. do you hate it?

Reply:
look at the readme preview and lmk I can change it back if you don't like it

Reply:
Nope it looks good, good shout